AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_BloomAdded.py

```diff
@@ -145,40 +366,569 @@ def AtomEditorComponents_Bloom_AddedToEntity():
+                'BicubicEnabled Override')) is False)
+
+        # 19. Enable the BicubicEnabled Override parameter.
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'BicubicEnabled Override'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.bicubicenabled_parameter_enabled,
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'BicubicEnabled Override')) is True)
+
+        # 20. Update the KernelSizeScale Override parameter to minimum value.
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'KernelSizeScale Override'), 0.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.kernelsizescale_override_min,
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'KernelSizeScale Override')) == 0.0)
+
+        # 21. Update the KernelSizeScale Override parameter to maximum value.
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'KernelSizeScale Override'), 1.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.kernelsizescale_override_max,
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'KernelSizeScale Override')) == 1.0)
+
+        # 22. Update the KernelSizeStage0 Override parameter to minimum value.
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'KernelSizeStage0 Override'), 0.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.kernelsizestage0_override_min,
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'KernelSizeStage0 Override')) == 0.0)
+
+        # 23. Update the KernelSizeStage0 Override parameter to maximum value.
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'KernelSizeStage0 Override'), 1.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.kernelsizestage0_override_max,
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'KernelSizeStage0 Override')) == 1.0)
+
+        # 24. Update the KernelSizeStage1 Override parameter to minimum value.
+        bloom_component.set_component_property_value(AtomComponentProperties.bloom(
+            'KernelSizeStage1 Override'), 0.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.kernelsizestage1_override_min,
+            bloom_component.get_component_property_value(AtomComponentProperties.bloom(
+                'KernelSizeStage1 Override')) == 0.0)
+
+        # 25. Update the KernelSizeStage1 Override parameter to maximum value.
```

Comment:
here and throughout: to reduce some of the steps since there are 73 steps (that's quite a bit lol) can we lump some of the comment labels together?
i.e. `# 25. Update the KernelSizeStage1 Override parameter to its minimum and maximum values.` then put all of the calls under each step under this same step

this would help reduce the test step count a bit since its quite daunting right now when viewing the script

Reply:
Cleaned up steps. I've combined everything that is a 2 step sequence (Enable/Disable // Min/Max), though I left comments in the code denoting which actions  are being taken in each step.